app/controllers/fitbit_profiles_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
     @fitbit_profiles = FitbitProfile
       .includes(:user)
       .order("#{sort_column} #{sort_direction}")
-      .paginate(page: params[:page], per_page: 20)
+      .paginate(page: params[:page], per_page: 15)
```

Comment:
I think this is unnecessary to change, what do you think? @philippbayer @gedankenstuecke @tsujigiri